### ISSUE TITLE
[BUG] H012 false positive

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#43]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3,7
 - djLint Version: 0.4.9
 - 
## Issue
Split from https://github.com/Riverside-Healthcare/djLint/issues/39#issuecomment-933095466 , this specific case still exists. (All the others I had are now gone, thx!)

## How To Reproduce
```html
<p>
        {% if activity.reporting_groups|length <= 0 %}<h3>{% trans 'General' %}</h3>{% endif %}
</p>
```
